stl/inc/ranges

```diff
@@ -1627,6 +1627,124 @@ namespace ranges {
         using all_t = decltype(all(_STD declval<_Rng>()));
     } // namespace views
 
+#if _HAS_CXX23
+    template <input_range _Vw>
+        requires view<_Vw>

```

Comment:
```suggestion
    template <view _Vw>
        requires input_range<_Vw>
```

consistency with the standard

Reply:
Here standard is inconsistent with itself, [`[range.as.rvalue.view]`](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p2446r2.html#class-template-as_rvalue_view-range.as.rvalue.view) says:
```c++
namespace std::ranges {
  template<input_range V>
    requires view<V>
  class as_rvalue_view : public view_interface<as_rvalue_view<V>>
  {
    V base_ = V(); // exposition only
//and so on
```

Reply:
GFDI, fair enough.